### DISCUSSION TITLE
Avoid/ stop getting Android Miscellaneous Notifications

### DISCUSSION THREAD

**Discussion by taatiq882** [invertase/react-native-firebase#6318] [Q&A]
I am working with cometchat and  I am connecting to cometchat push notification services using this line of code 
 "wait CometChat.registerTokenForPushNotification(FCM_TOKEN);"
 
 By just this line I automatically start receiving notifications with the Miscellaneous channel although I haven't set up the background message handler.   So when I set up a background message handler with notifee I get two notifications of the same message.  

`messaging().setBackgroundMessageHandler(async remoteMessage => {
  console.log('Message handled in the background!', remoteMessage);

  const channelId = await notifee.createChannel({
    id: 'default',
    name: 'Default Channel',
  });

      await notifee.displayNotification({
        title: remoteMessage.data.title,
        body: remoteMessage.data.alert,
        android: {
          channelId,
          smallIcon: 'ic_notification', 
          color: CustomColors.SECONDARY,
        },
      });
      
});`

**Comment by mikehardy** [comment]
So it sounds like you want to look at the CometChat API and figure out how/where they let you turn off their default / built-in notifications?

**Comment by EdGitLink** [comment]
I ran into the same thing and just found the solution.
Go into your cometchat console, into extensions, into Push Notifications.
Under Payload settings, turn off "Notification" key for what you're using (I assume react native?)

This will prevent firebase from showing the default notification and you can then use notifee to generate the notification.

This is probably too late for you, but hoping it helps the next person...
